workers/unity/Packages/com.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -195,11 +190,24 @@ private static string[] ConstructArgs(string projectPath, string schemaCompilerP
             baseArgs.Add($"--native-output-dir=\"{toolsConfig.CodegenOutputDir}\"");
             baseArgs.Add($"--schema-path=\"{toolsConfig.SchemaStdLibDir}\"");
 
+            // Add user defined schema directories
             foreach (var schemaSourceDir in toolsConfig.SchemaSourceDirs)
             {
-                baseArgs.Add($"--schema-path=\"{schemaSourceDir}\"");
+                if (Directory.Exists(schemaSourceDir))
```

Comment:
I wonder if it's better to show an error in the configuration UI rather than silently ignoring invalid inputs here.